test/e2e/framework.go

```diff
@@ -307,17 +307,40 @@ func (data *TestData) createVCNamespace(namespace string) error {
 	}()
 
 	svID, _ := data.vcClient.getSupervisorID()
+	storagePolicyID, _ := data.vcClient.getStoragePolicyID()
```

Comment:
Will content library be heavy load or have performance for Namespace, can we add a parameter in `createVCNamespace` to add storage policy and content library only if they are needed in specific test cases?